GOVERNANCE-subprojects.md

```diff
@@ -0,0 +1,172 @@
+# [TODO:PROJECTNAME] Project Governance
+
+This is an example governance document for projects that consist of a group
+of smaller, semi-independent projects.  See [GOVERNANCE.md](/GOVERNANCE.md)
```

Comment:
On reading this again with your changes, I think some of my earlier confusion stems from how we are using project and subproject. I recommend consistently using "project" to refer to the collection (like Konveyor Project) and always say "subproject" when we refer to an individual project within the collection. I find myself getting confused about whether something refers to the group or subproject.

In this case: "... semi-independent subprojects"

Reply:
This has been updated across the file, including replacing 40 instances of "project" with "subproject".